### PR TITLE
Automating GNF message

### PR DESCRIPTION
**Pull request by DarrylBrooks97**

### CODE REVIEW

File: utils/index.ts [hunk 1]
```diff
@@ -11,3 +11,12 @@ export const formatSocial = (name: keyof typeof socials, value: string) => {
...
     link?.includes('github') ? 'github' : link?.includes('linkedin') ? 'linkedin' : 'twitter';
+
+export const addLeadingZero = (x: number) => (x.toString().length < 2 ? `0${x}` : x);
+
+export const getFormattedDate = () => {
+    const dateObj = new Date();
+    return `${dateObj.getUTCFullYear()}-${addLeadingZero(
+        dateObj.getUTCMonth() + 1,
+    )}-${addLeadingZero(dateObj.getUTCDate())}`;
+};
```
Comment (gt-codes) [inline utils/index.ts]: Use the `date-fns` [library](https://date-fns.org/)

**Comment by DarrylBrooks97** [comment]
POC: https://www.loom.com/share/42f39c1edf464d56b13cf4ebf7aff5ca

File: yarn.lock [hunk 2]
```diff
@@ -1161,7 +1161,16 @@ cross-spawn@^7.0.2, cross-spawn@^7.0.3:
...
+<<<<<<< HEAD
+date-fns@^2.29.2:
+  version "2.29.2"
+  resolved "https://registry.yarnpkg.com/date-fns/-/date-fns-2.29.2.tgz#0d4b3d0f3dff0f920820a070920f0d9662c51931"
+  integrity sha512-0VNbwmWJDS/G3ySwFSJA3ayhbURMTJLtwM2DTxf9CWondCnh6DTNlO9JgRSq6ibf4eD0lfMJNBxUdEAHHix+bA==
+
+debug@^4.1.1, debug@^4.3.2:
+=======
 debug@^4.1.0, debug@^4.1.1, debug@^4.3.2:
+>>>>>>> 6b886c9ae2cc09e3f01722c1821a98467613c176
```
Comment (gt-codes) [inline yarn.lock]:
```suggestion
date-fns@^2.29.2:
  version "2.29.2"
  resolved "https://registry.yarnpkg.com/date-fns/-/date-fns-2.29.2.tgz#0d4b3d0f3dff0f920820a070920f0d9662c51931"
  integrity sha512-0VNbwmWJDS/G3ySwFSJA3ayhbURMTJLtwM2DTxf9CWondCnh6DTNlO9JgRSq6ibf4eD0lfMJNBxUdEAHHix+bA==

debug@^4.1.1, debug@^4.3.2:
```

**Review by gt-codes — CHANGES_REQUESTED** [review]
I love the loom! Much appreciated. Just accept the `yarn.lock` suggestion and it's GTG

**Pull request merged**